apsapi/__init__.py

```diff
@@ -0,0 +1,80 @@
+import httplib
+import json
+import ssl
+import urllib2
+from urlparse import urljoin
+
+
+class HTTPSClientAuthHandler(urllib2.HTTPSHandler, object):
+    def __init__(self, key, cert, context=None):
+        super(HTTPSClientAuthHandler, self).__init__(context=context)
+        self.key = key
+        self.cert = cert
+
+    def https_open(self, req):
+        # Rather than pass in a reference to a connection class, we pass in
+        # a reference to a function which, for all intents and purposes,
+        # will behave as a constructor
+        return self.do_open(self.get_connection, req)
+
+    def get_connection(self, host, timeout=300):
+        return httplib.HTTPSConnection(host, key_file=self.key, cert_file=self.cert, context=self._context)
+
+
+class API(object):
+    def __init__(self, url, use_unverified_context=False):
+        """
+        Takes something like the following argument as input:
+        url = 'https://a.bvt.aps.sw.ru:6308'
+        """
+        self.url = url
+        self.use_unverified_context = use_unverified_context
+
+    def call(self, verb, path, headers=None, data=None, cert=None):
+        data = json.dumps(data)
```

Comment:
if data is `None` then the resulting json would be `null` which is a weird body for REST. Use smth like

```python
if not data:
    data = {}
```